src/pagespeed.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
 
-module.exports = async function(page, apiKey) {
-	let url = `https://www.googleapis.com/pagespeedonline/v5/runPagespeed?url=${page}&key=${apiKey}&category=PERFORMANCE&category=ACCESSIBILITY`;
+module.exports = async function(page, apiKey, viewport) {
```

Comment:
Could we decouple the function interface and the api call arguments? Perhaps using exposed constants exposed from the module?

Can we either provide a sensible default or use an input parameter guard to throw an exception when an invalid viewport is passed?

Reply:
We could take the stuff we did in the other PR and apply it here?